_data/sidebars/main.yml

```
@@ -76,6 +76,8 @@ subitems:
         url:  /showcase/covid19-galaxy
       - title: Linked Cohort Data
         url: /showcase/linked-cohort-data
+      - title: Prevalence study of SARS-CoV-2 in Estonia: lessons learned
```

```suggestion
      - title: "Prevalence study of SARS-CoV-2 in Estonia: lessons learned"
```
When using a colon inside a string, you need it in quotes to have valid YAML